.docs/user-guide/usage.md

```diff
@@ -0,0 +1,210 @@
+# Usage
+Report status, Initiate storage operations
+
+---
+
+## Summary
+General format:
+
+```sh
+$ rexray [option flags] [command]
+```
+
+Get general or command specific help
+
+```sh
+$ rexray --help
+```
+
+```sh
+$ rexray [command] --help
+```
+
+## Global Option Flags
+These can be invoked with all commands
+
+| Flag | Alternate form | Description |
+| --- | --- | --- |
+| `-c` | `--config=""` | Path to a custom REX-Ray configuration file |
+| `-h` | `--host=""` | designate libStorage host to execute command |
+| `-l` | `--logLevel=""` | error, warn, info, debug |
+| `-s` | `--service=""` | designate specific libStorage service command pertains to |
+| `-v` | `--verbose=true` | emit verbose information to console |
+
+## Commands
+| Command | Description |
+| --- | --- |
+| version | return REX-Ray and libStorage versions |
```

Comment:
Hi @cantbewong,

Please ensure that all commands in the `Command` column are formatted as code.